### DISCUSSION TITLE
Do we prefer 0-255 rgb values as default?

### DISCUSSION THREAD

**Discussion by slmjkdbtl** [replit/kaboom#158] [General]
There has been some efforts in community to enable 0-255 rgb colors. Is it what everyone prefer as default? 

Same goes with radians vs degrees. Is degrees by default better for kaboom (as a beginner facing tool)?

**Comment by lajbel** [comment]
I prefer rgb 255 and degrees, I think they are more understandable for users who come from other engines or are new
